app.js

```diff
@@ -2,22 +2,47 @@ const express = require('express');
 const {
   getCategories,
   getReviews,
+  getReviewById
 } = require('./controllers/games.js');
 
 const app = express();
 app.use(express.json());
```

Comment:
app.use(express.json());
We don't need this line of code yet, so this line should not exist. Delete this until we need it